scripts/us_epa/parent_company/testdata/expected/svobs/SVObs.tmcf

```diff
@@ -2,8 +2,8 @@ Node: E:EpaParentCompanyStatVarObs->E0
 typeOf: dcs:StatVarObservation
 observationAbout: C:EpaParentCompanyStatVarObs->parent_company_dcid
 variableMeasured: C:EpaParentCompanyStatVarObs->sv_measured
-observationPeriod: "C:EpaParentCompanyStatVarObs->obs_period"
+observationPeriod: C:EpaParentCompanyStatVarObs->obs_period
```

Comment:
Filed https://github.com/datacommonsorg/import/issues/111 so dc-import would've caught this.

Reply:
Thanks! I included this in the updated description as well.